src/Microsoft.Graph.Core/Extensions/ITokenValidableExtension.cs

```diff
@@ -41,28 +41,46 @@ public static class ITokenValidableExtension
                 throw new ArgumentNullException(nameof(tenantIds));
             if (appIds == null || !appIds.Any())
                 throw new ArgumentNullException(nameof(appIds));
+            if(!issuerFormats?.Any() ?? true)
```

Comment:
```suggestion
            if(!(issuerFormats?.Any() ?? false))
```
I'm worried that this one will evaluate the other way around if issuerFormarts is null

Reply:
I hear you